docs/run_test/platform.rst

```diff
@@ -179,6 +180,9 @@ deployment.
   run all test cases. Notice, there are some features are conflict by natural,
   so some test cases may not be picked up. This setting is useful to force run
   perf tests on not designed VM sizes.
+* **osdisk_size_in_gb** is used to specify the minimum size of the OS disk. The size
+  of disk may be larger depending on other requirements set by vhd, shared image gallery
+  or marketplace image.
```

Comment:
it is the size of the os disk, not `the minimum`, and when the specified os disk size is less than the default disk size of the  image/vhd/shared image, then default os size will be used. also we need mention when osdisk_size_in_gb will be used, and the range, refer to `https://learn.microsoft.com/en-us/azure/virtual-machines/linux/expand-disks?tabs=ubuntu`

Reply:
Updated, please check